tests/apis.js

```diff
@@ -0,0 +1,430 @@
+    t.true(item.hasOwnProperty('key_slug'), 'Contains Key Slug');
+    t.true(item.hasOwnProperty('meta'), 'Contains Meta');
+    t.false(item.hasOwnProperty('attributes'), 'Does not contain attributes');
+    t.is(item.meta.url, `/api/types/${item.type_slug}/${item.id}`, 'URL points to full content item');
+  });
+});
+
+test('Utils: Format Results - Attributes', t => {
+  const formatted = apiUtils.format(content.slice(0, 9), true);
+
+  formatted.forEach(item => {
+    t.true(item.hasOwnProperty('id'), 'Contains ID');
+    t.true(item.hasOwnProperty('type'), 'Contains Type');
+    t.true(item.hasOwnProperty('type_slug'), 'Contains Type Slug');
+    t.true(item.hasOwnProperty('key'), 'Contains Key');
+    t.true(item.hasOwnProperty('key_slug'), 'Contains Key Slug');
+    t.false(item.hasOwnProperty('meta'), 'Does not contain Meta');
+    t.true(item.hasOwnProperty('attributes'), 'Contains attributes');
+  });
+});
+
+test('Utils: Organize - Default', t => {
+  const actual = apiUtils.organize({});
+  const expected = {
+    sort: {
+      by: 'key',
+      dir: 'asc',
+    },
+    page: {
+      offset: 0,
+      limit: 30,
+      page: 1,
+    },
+  };
+
+  t.deepEqual(actual, expected, 'Returns defaults');
+});
+
+test('Utils: Organize - Lookup', t => {
+  const actual = apiUtils.organize({}, [
+    'foo',
+    'bar',
+    'baz',
+  ]);
+  const expected = {
+    sort: {
+      by: 'foo',
+      dir: 'asc',
+    },
+    page: {
+      offset: 0,
+      limit: 30,
+      page: 1,
+    },
+  };
+
+  t.deepEqual(actual, expected, 'Returns defaults');
+});
+
+test('Utils: Organize - Custom', t => {
+  const actual = apiUtils.organize({
+    sort: 'type',
+    sort_dir: 'desc', // eslint-disable-line camelcase
+    page: 2,
+    per_page: 50, // eslint-disable-line camelcase
+  });
+
+  const expected = {
+    sort: {
+      by: 'type',
+      dir: 'desc',
+    },
+    page: {
+      offset: 50,
+      limit: 50,
+      page: 2,
+    },
+  };
+
+  t.deepEqual(actual, expected, 'Returns defaults');
+});
+
+test('Utils: Organize - Wrong', t => {
+  const actual = apiUtils.organize({
+    sort: 'foo',
+    sort_dir: 'banana', // eslint-disable-line camelcase
+    page: 0,
+    per_page: 200, // eslint-disable-line camelcase
+  });
+
+  const expected = {
+    sort: {
+      by: 'key',
+      dir: 'asc',
+    },
+    page: {
+      offset: 0,
+      limit: 100,
+      page: 1,
+    },
+  };
+
+  t.deepEqual(actual, expected, 'Returns defaults');
+});
+
+test('Utils: Organize - Wrong Pages', t => {
+  const actual = apiUtils.organize({
+    page: -1,
+    per_page: -1, // eslint-disable-line camelcase
+  });
+
+  const expected = {
+    sort: {
+      by: 'key',
+      dir: 'asc',
+    },
+    page: {
+      offset: 0,
+      limit: 1,
+      page: 1,
+    },
+  };
+
+  t.deepEqual(actual, expected, 'Returns defaults');
+});
+
+test('Utils: Page - First', t => {
+  const organized = apiUtils.organize({});
```

Comment:
This included `{}` object could be skipped if there was error checking on `organize`